test/helpers.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: MIT */
+/*
+ * Description: Helpers for tests.
+ */
+#ifndef HELPERS_H
```

Comment:
Should probably make that LIBURING_HELPERS_H just to avoid a name that's too generic and might conflict.